data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/impl/RecordDispatcherImpl.java

```diff
@@ -333,41 +333,44 @@ private Tags getTags(final ConsumerRecordContext recordContext) {
     );
   }
 
-  private static void logError(
+  private void logError(
     final String msg,
     final KafkaConsumerRecord<Object, CloudEvent> record,
     final Throwable cause) {
 
     if (logger.isDebugEnabled()) {
-      logger.error(msg + " {} {} {} {} {}",
+      logger.error(msg + " {} {} {} {} {} {}",
         keyValue("topic", record.topic()),
         keyValue("partition", record.partition()),
         keyValue("headers", record.headers()),
         keyValue("offset", record.offset()),
         keyValue("event", record.value()),
```

Comment:
I am supportive on adding the consumer group.

but perhaps we can set the above to `trace` as well, since we do already log the value. I know - not introduced by you :sweat_smile:

Reply:
I don't mind, I'd rather make the updates if it is something that I am touching on. Rather see less tech debt if we can make the improvements while we are in the area!

I'll look to make the debug->trace updates later this afternoon, thanks for the feedback!

Reply:
Took longer than I wanted, but debug -> trace is added now :+1: